bedrock/firefox/templates/firefox/mobile.html

```diff
@@ -331,6 +331,9 @@ <h3>{{ _('Lightweight') }}</h3>
         <li class="android">
           {{ google_play_button(alt_href=settings.GOOGLE_PLAY_FIREFOX_FOCUS_LINK, anchor_attributes={'data-link-type': 'download', 'data-download-os': 'Android', 'id': 'playStoreLink'}) }}
         </li>
+        <li class="android-apk">
```

Comment:
For l10n, you should wrap this whole `<li>` in an `l10n_has_tag` conditional. There are many of these in the code base, including on the [firefox home page](https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/home.html#L75).

You can make up your own appropriate tag name for the lang file. I'd probably choose like `q12018_android_apk_link`. (I personally prefer putting some kind of timestamp in the tag name so when we look at it in 2 years we can say, "Hey, that tag is two years old. Can't we remove it?")